firefox-ios/Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -67,13 +67,17 @@ class ClearPrivateDataTableViewController: ThemedTableViewController {
         self.profile = profile
         self.tabManager = tabManager
 
-        super.init()
+        super.init(windowUUID: tabManager.windowUUID)
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    private func currentTheme() -> Theme {
+        return themeManager.currentTheme(for: windowUUID)
+    }
```

Comment:
seems like we are calling this in multiple places, would it be beneficial to call this less; only during initialization or theme change?

Reply:
Yeah this is a problem I was noticing all over the place, we spam calls to `themeManager.currentTheme()`. I tried to DRY out some of that during this refactor but it's not really in the scope of 8313 and so I think some of that will need to be cleaned-up in a follow-up. It is definitely on my radar though.